l10n_ch_bank_statement_import_postfinance/models/postfinance_file_parser.py

```diff
@@ -64,6 +64,52 @@ def parse_transaction_details(self, ns, node, transaction):
         super(XMLPFParser, self).parse_transaction_details(
             ns, node, transaction)
 
+    # Copied from bank-statement-import/account_bank_statement_import_camt/
+    # models/parser.py
+    def parse_entry(self, ns, node):
```

Comment:
This is not good. This class is supposed to inherit the parser, so why would you copy the method?